### PR TITLE
feat: implemented scopes retrieval

### PR DESCRIPTION
**Pull request by ayZagen**

### CODE REVIEW

File: src/store/create-store-module.js [hunk 1]
```diff
@@ -84,6 +85,9 @@ export default (oidcSettings, storeSettings = {}, oidcEventListeners = {}) => {
     oidcAccessTokenExp: (state) => {
       return tokenExp(state.access_token)
     },
+    oidcScopes: (state) => {
```
Comment (ayZagen) [inline src/store/create-store-module.js]: actually we don't need getters for only retrieving the state, but I tried to follow your code

Comment (perarnborg) [reply src/store/create-store-module.js]: Nice. I've opted to use getters more for the future, if some middle layer features would need to be applied. Thanks for sticking with that!

**Comment by perarnborg** [comment]
Thank you @ayZagen for your contribution! I will take a closer look soon, but looks good at first glance!

**Comment by perarnborg** [comment]
@ayZagen This change is merged and published in v2.0.1. Thanks for your contribution!

**Comment by ayZagen** [comment]
Thank you for the library

**Pull request merged**